config/env_blank.json

```diff
@@ -13,9 +13,14 @@
   "CANVAS_URL": "",
   "CANVAS_TOKEN": "",
   "NUM_ASYNC_WORKERS": 8,
-  "ZOOM_TOKEN": "<ADD THIS TOKEN>",
-  "ZOOM_BASE_URL": "<ADD zoom.us URL>",
-  "ZOOM_EARLIEST_FROM": "YYYY-MM-DD",
+  "ZOOM_CONFIG": {
+    "1": {
```

Comment:
Could this be an `int`?

Reply:
Also, what do you think the likelihood is of someone just mixing these up and putting one account that should be "1" as "2"?

Reply:
The number and order doesn't matter, this is the value that goes into a column in the database. I didn't know what Chris wanted so it's currently a string. If could be "um-zoom" or "um-zoom-health" as a VARCHAR unless we're doing a mapping.

Reply:
Yeah, I'm concerned that someone will run it one way, and then someone else will do it and run it another.

Reply:
I think this will need to be a separate ticket, probably can be done on #31.
 
When we create the DB tables for this we should probably have a mapping table. I feel like these are fine as an int and probably will need some way to define the mapping (probably via migration), just waiting to hear back from Chris.

Reply:
I added this as a task to #31